code/go/0chain.net/blobbercore/challenge/challenge.go

```diff
@@ -131,7 +135,11 @@ func processAccepted(ctx context.Context) {
 }
 
 func validateChallenge(swg *sizedwaitgroup.SizedWaitGroup, challengeObj *ChallengeEntity) {
```

Comment:
for this to be called we should just be playing with `challenge_response.frequency`
make it shorter if we need it to be called frequently